src/routes/deploy.ts

```diff
@@ -247,14 +263,17 @@ export function setupDeployRoutes(app: Application) {
                 }
             }
             logAndSave('info', 'Successfully deployed all containers');
-            res.status(200).send(logOutput);
+            c.text(logOutput, {
+                status: 200,
+            });
         } catch (error) {
             log('error', `Failed to deploy containers: ${error.message}`);
-            if (!res.headersSent) {
-                res.status(500).send(
-                    'Internal server error. View logs for more information.',
-                );
-            }
+            // if (!res.headersSent) {
+            // TODO: What is the hono equivalent of this?
```

Comment:
?

Reply:
Sollte in Hono auch nicht benötigt werden. Aber du musst glaube ich die Reponse immer returnen?
Also `return c.text(...)`?